### PR TITLE
x11-misc/splatmoji: new package

### PR DESCRIPTION
**Pull request by WPettersson**
splatmoji is a tool that lets users quickly look up and input emoji,
emoticons and kaomoji via a pop-up menu.

Signed-off-by: William Pettersson <william@ewpettersson.se>
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

**Comment by hydrapolic** [comment]
Can you please also create a snapshot version so that it's not a constantly moving target?

**Comment by hydrapolic** [comment]
Ideally upstream creates releases, but i don't see any yet.

**Comment by WPettersson** [comment]
Just checking, how do I name a snapshot version in this scenario? I'm used to splatmoji-1.0.0_pre20190120 if it's just before release 1.0.0, but I don't know what any next release would be. Should it just be splatmoji_20190120.ebuild ?

File: x11-misc/splatmoji/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<longdescription>
```
[the remark this answers is not in the document]
Comment (WPettersson) [reply x11-misc/splatmoji/metadata.xml]: Moved maintainer first.

File: x11-misc/splatmoji/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		splatmoji is a tool that lets users quickly look up and input emoji, emoticons and kaomoji via a pop-up menu.
```
[the remark this answers is not in the document]
Comment (WPettersson) [reply x11-misc/splatmoji/metadata.xml]: Started with capital S now, and wrapped long line

**Comment by WPettersson** [comment]
Managed to get an actual upstream release, so we now have version 1.0 as well as the live ebuild.

I do have one comment - splatmoji is essentially a bash script. Is it okay to assume that a Gentoo install will always have bash, or should I DEPEND on bash as well?

**Comment by hydrapolic** [comment]
> Managed to get an actual upstream release, so we now have version 1.0 as well as the live ebuild.
> 
> I do have one comment - splatmoji is essentially a bash script. Is it okay to assume that a Gentoo install will always have bash, or should I DEPEND on bash as well?

Good point, RDEPEND on app-shells/bash should be fine.

**Comment by juippis** [comment]
> Is it okay to assume that a Gentoo install will always have bash, or should I DEPEND on bash as well?

Yes, it's part of BASE packages for every profile, and it's not replacable. I wouldn't mind if you had it there, but strictly you don't need it in *DEPEND.

**Comment by juippis** [comment]
I was wrong btw, instead of `_p` it should be `_pre`. Sorry about that. 

https://github.com/gentoo/gentoo/pull/10222#discussion_r230669503
https://devmanual.gentoo.org/ebuild-writing/file-format/index.html

**Comment by juippis** [comment]
Hey, I know its been forever, but are you still around and willing to maintain this in Gentoo? 
I can now merge it, but only if it has a maintainer.

**Comment by WPettersson** [comment]
Hi, yes still around, and still happy to maintain.

**Pull request closed without merging**